src/main/java/org/recap/service/executor/datadump/DeletedDataDumpExecutorService.java

```diff
@@ -43,10 +44,10 @@ public void populateSearchRequest(SearchRecordsRequest searchRecordsRequest, Dat
         searchRecordsRequest.setDeleted(true);
         if(StringUtils.isNotBlank(dataDumpRequest.getDate()) && !onlyOrphan) {
             searchRecordsRequest.setFieldName(RecapCommonConstants.ITEM_LASTUPDATED_DATE);
-            searchRecordsRequest.setFieldValue(getFormattedDateString(dataDumpRequest.getDate(), dataDumpRequest.getToDate()));
+            searchRecordsRequest.setFieldValue(DateUtil.getFormattedDateString(dataDumpRequest.getDate(), dataDumpRequest.getToDate()));
```

Comment:
When you have a criteria to be checked in both the scenarios, make it to the top level. In your case dataDumpRequest.getDate is checked in both criteria. If it is failed in first, for sure else also fails, but we are unnecessarily processing a line of code, if it is or condition well and good, but it is not.

No need to change it now, implemented it in your future changes.